datasets/standvastgoed/dataset.json

```diff
@@ -1,7 +1,7 @@
 {
   "type": "dataset",
   "id": "standVastgoed",
-  "version": "0.0.4",
+  "version": "0.0.5",
```

Comment:
Bij een bump in de versie is het eigenlijk de bedoeling de table uit te splitsen in een apart bestand en een extra bestand voor de nieuwe versie aan te maken (zie bijv. de bag dataset voor voorbeelden). Nu is deze dataset nog in een soort pre-release stadium, dus wellicht hier nog overkill. Vraag is dan wel of het versienummer dan wel gebumped moet worden.

Reply:
Overigens, voor dat uitsplitsen en bumpen-bij-table-in-los-bestand zijn er twee cli commandootjes in de repo `dataset-split` en `table-bump`.